postgres/stack.yml

```
@@ -1,11 +1,19 @@
 # Use postgres/example user/password credentials
-version: '3.1'
+version: '3.9'
 
 services:
 
   db:
     image: postgres
     restart: always
+    # set shared memory limit when deploy swarm stack
+    tmpfs:
+      - /tmp:size=128mb
+    volumes:
+      - type: tmpfs
+        target: /dev/shm
+    # set shared memory limit when deploy through docker-compose
+    # shm_size: 128mb
```

We should swap the default to `shm_size` and then comment for the swarm stack version since we plan to rename them all back to `docker-compose.yml` at some point.

```suggestion
    # set shared memory limit when using docker-compose
    shm_size: 128mb
    # or set shared memory limit when deploy via swarm stack
    #tmpfs:
    #  - /dev/shm:size=128m
```